src/Components/ContactsListComponents/ContactImage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image } from 'react-native';
+
+const ContactImage = ({ contacts }) => {
+  const nameAndSurname = contacts.item.name.split(' ');
+  const userInitials = nameAndSurname[0][0] + nameAndSurname[1][0];
+  const contactPictureAdress = contacts.item.photo;
+  const userInitialsComponents = (
+    <View style={styles.initialsContainer}>
+      <Text style={styles.initialsTextStyle}>{userInitials}</Text>
+    </View>
+  );
+  let userPictureComponents = (
+    <View>
+      <Image
+        style={styles.pictureStyle}
+        source={require('../ContactsListComponents/ContactsImage/Rectangle60.png')}
+      />
+    </View>
```

Comment:
Please make constant variables in "const",
Create small components out of ContactImage.
1, <Initials />, and pass name. write logic inside the component.
2, <Avatar /> and pass image prop.